### PR TITLE
Add a mean to set the local address in the perf_client

### PR DESCRIPTION
**Pull request by lu-zero**

### CODE REVIEW

File: perf/src/bin/perf_client.rs [hunk 1]
```diff
@@ -48,6 +48,9 @@ struct Opt {
     /// Receive buffer size in bytes
     #[structopt(long, default_value = "2097152")]
     recv_buffer_size: usize,
+    /// Specify the local ip
+    #[structopt(long)]
+    local_ip: Option<IpAddr>,
```
Comment (Ralith) [inline perf/src/bin/perf_client.rs:53]:
```suggestion
    /// Specify the local address
    #[structopt(long)]
    local_addr: Option<SocketAddr>,
```

File: perf/src/bin/perf_client.rs [hunk 2]
```diff
@@ -79,11 +82,14 @@ async fn run(opt: Opt) -> Result<()> {
...
-        Ipv6Addr::UNSPECIFIED.into()
-    };
+    let bind_addr = opt.local_ip.unwrap_or_else(|| {
+        if addr.is_ipv4() {
+            Ipv4Addr::UNSPECIFIED.into()
+        } else {
+            Ipv6Addr::UNSPECIFIED.into()
+        }
+    });
+    info!("Local addr {:?}", bind_addr);
```
Comment (Ralith) [inline perf/src/bin/perf_client.rs:92]:
```suggestion
    info!("local addr {:?}", bind_addr);
```

**Review by Ralith — CHANGES_REQUESTED** [review]
Thanks! Might as well make this a little more flexible, I think.

File: perf/src/bin/perf_client.rs (hunk 1 above)
Comment (lu-zero) [reply perf/src/bin/perf_client.rs:53]: I'll update it to use a socketaddr soon :)

**Review by djc — APPROVED** [review]

**Pull request merged**